test/net/imap/fixtures/response_parser/body_structure_responses.yml

```
@@ -433,6 +433,77 @@
         4383638 NIL (\"attachment\" (\"filename\" \"test.xml\")) NIL NIL) \"mixed\"
         (\"boundary\" \"001a1137a5047848e405157ddaa3\") NIL))\r\n"
 
+  test_bodystructure_extra_space:
```

Would you mind adding a comment to document this in the test fixture?

```suggestion
  test_bodystructure_extra_space:
    :comments: |
      [GH-271] Responses from some IMAP servers contained extra space like this.
      The fix has been used since 2016, seems to be easy, and doesn't seem to
      cause any harm.
```

I just copied over some of your issue description, but please feel free to rewrite this to whatever you think would be useful.  Even better, can can share _which_ servers (the name of an ISP or email host, or a hostname, or a product name and version) need this workaround?  Thanks!

---

I tried to find any traces of why we've added this workaround, but couldn't find anything. I will update your description as I found that originally this workaround was introduced in 2014. So it's somewhat reasonable why I cannot find any traces of why we have this :)

---

That's all right.  Thanks for looking into it.